wrappers/clang-target-wrapper.c

```diff
@@ -292,6 +295,22 @@ int _tmain(int argc, TCHAR* argv[]) {
     } else if (!_tcscmp(arch, _T("aarch64"))) {
         // Dwarf is the default here.
     }
+    
+    if (!_tcscmp(target_os, _T("mingw32uwp"))) {
```

Comment:
Technically, `target_os` could be NULL here, so `if (target_os && !_tcscmp())` probably is safest.